agent/acs/handler/acs_handler_test.go

```diff
@@ -518,10 +518,14 @@ func TestHandlerReconnectDelayForInactiveInstanceError(t *testing.T) {
 		}).Return(fmt.Errorf("InactiveInstanceException:")),
 		mockWsClient.EXPECT().Connect().Do(func() {
 			reconnectDelay := time.Now().Sub(firstConnectionAttemptTime)
+			reconnectDelayTime := time.Now()
 			t.Logf("Delay between successive connections: %v", reconnectDelay)
-			if reconnectDelay < inactiveInstanceReconnectDelay {
-				t.Errorf("Reconnect delay %v is not less than inactive instance reconnect delay %v", reconnectDelay, inactiveInstanceReconnectDelay)
-			}
+			timeSubFuncSlopAllowed := 2 * time.Millisecond
+			// On windows platform, we found issue with time.Now().Sub(...) reporting 199.9989 even
+			// after the code has already waited for time.NewTimer(200)ms.
+			assert.WithinDuration(t, reconnectDelayTime, firstConnectionAttemptTime.Add(inactiveInstanceReconnectDelay), timeSubFuncSlopAllowed,
+				"Reconnect delay %v is expected within %v of inactive instance reconnect delay %v",
```

Comment:
You mean
1) I should remove comment (line 524 to line 525)?
or
2) rewording remove line 527?
thanks